cmake/Modules/GnuradioConfig.cmake.in

```diff
@@ -62,6 +62,9 @@ set(GR_RPCSERVER_ENABLED    @GR_RPCSERVER_ENABLED@)
 set(ENABLE_CTRLPORT_THRIFT  @ENABLE_CTRLPORT_THRIFT@)
 set(THRIFT_FOUND            @THRIFT_FOUND@)
 set(GR_RPCSERVER_THRIFT     @GR_RPCSERVER_THRIFT@)
+if(@THRIFT_FOUND@)
+  find_package(THRIFT)
```

Comment:
No, please use `add_dependency()`

Reply:
Even put this in `gnuradio-runtime/lib/CMakeLists.txt` the `GR_LIBRARY_FOO` has a optional argument which will add `find_dependency()` calls to the target configuration.

Reply:
I don't understand your second comment. This line is directly stolen from runtime/lib/CMakeLists.txt, including the fact that it's `find_package`, not `_dependency`.

Reply:
also, just tested it: `find_dependency` does **not** work; OOTs refuse to build (like if I had not attempted to find thrift at all):

<pre>CMake Error at lib/CMakeLists.txt:37 (add_library):
  Target &quot;gnuradio-testi&quot; links to target &quot;Thrift::thrift&quot; but the target was
  not found.  Perhaps a find_package() call is missing for an IMPORTED
  target, or an ALIAS target is missing?


CMake Error at /home/marcus/.usrlocal/lib64/cmake/gnuradio/UseSWIG.cmake:573 (add_library):
  Target &quot;testi_swig&quot; links to target &quot;Thrift::thrift&quot; but the target was not
  found.  Perhaps a find_package() call is missing for an IMPORTED target, or
  an ALIAS target is missing?
Call Stack (most recent call first):
  /home/marcus/.usrlocal/lib64/cmake/gnuradio/GrSwig.cmake:133 (swig_add_library)
  swig/CMakeLists.txt:51 (GR_SWIG_MAKE)


CMake Error at lib/CMakeLists.txt:37 (add_library):
  Target &quot;gnuradio-testi&quot; links to target &quot;Thrift::thrift&quot; but the target was
  not found.  Perhaps a find_package() call is missing for an IMPORTED
  target, or an ALIAS target is missing?
</pre>

Reply:
No need to specifically emphasize. Rather look at the generated Config. `find_package()` usage in <Package>Config.cmake is not advised: https://cmake.org/cmake/help/v3.0/module/CMakeFindDependencyMacro.html

Reply:
Obviously this is the wrong place in the file anyway. Has to come before `include(../gnuradio-runtimeTargets.cmake)`. But there is a better way anyway.